tests/Resources/keywords_avdelning_mottagning.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Documentation
+Library  SeleniumLibrary
+
+*** Keywords ***
+
+Expandera Kontaktblock Test
+
+    ${width}	${height} =	            Get Element Size	css:#main-content > div > div.department-wrapper > div.main-wrapper.department-one > div.row > div > div > div.kivcontact__column-container
```

Comment:
I wonder if it's possible to make these selectors a little shorter while still targeting a unique element...
`.department-one div.kivcontact__column-container` is maybe enough?

Reply:
Alright, will give it a try!